engine/runtime/action-source/src/file/reader/runner.rs

```diff
@@ -63,6 +63,13 @@ pub enum FileReader {
         #[serde(flatten)]
         common_property: FileReaderCommonParam,
     },
+    /// # Shapefile
+    Shapefile {
+        #[serde(flatten)]
+        common_property: FileReaderCommonParam,
+        #[serde(flatten)]
+        property: shapefile::ShapefileReaderParam,
+    },
```

Comment:
Could you implement this as an independent `ShapefileReader` instead of as part of `FileReader`?

We are planning to deprecate the implementation in `FileReader`, and the plan is to make all of them independent `Reader`s in the same way.

For reference, please take a look at [this PR](https://github.com/reearth/reearth-flow/pull/1192/files).